<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PixelTForm</class>
 <widget class="QDialog" name="PixelTForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set pixel transformation</string>
  </property>
  <widget class="QSpinBox" name="origPixelBox1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="origPixelBox4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="origPixelBox2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>85</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="origPixelBox3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>170</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="newPixelBox4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="newPixelBox2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>85</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="newPixelBox3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>170</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="newPixelBox1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="origLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Original pixel value&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="origLabel_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;New pixel value&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyNeg">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply negative</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>origPixelBox1</tabstop>
  <tabstop>origPixelBox2</tabstop>
  <tabstop>origPixelBox3</tabstop>
  <tabstop>origPixelBox4</tabstop>
  <tabstop>newPixelBox1</tabstop>
  <tabstop>newPixelBox2</tabstop>
  <tabstop>newPixelBox3</tabstop>
  <tabstop>newPixelBox4</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
